webclient/services/mailer.py

```diff
@@ -184,9 +184,13 @@ def send_eu06(cls, user: 'uzivatel.models.User', groups):
         IDENT_CELY = 'E-U-06'
         logger.debug("services.mailer.send_eu06", extra={"ident_cely": IDENT_CELY})
         notification_type = uzivatel.models.UserNotificationType.objects.get(ident_cely=IDENT_CELY)
-        roles = ", "
-        roles = roles.join([group.name for group in groups])
-        roles = roles.rstrip(', ')
+        if groups[0] is not None:
+            logger.debug(groups)
+            roles = ", "
+            roles = roles.join([group.name for group in groups])
+            roles = roles.rstrip(', ')
```

Comment:
Je tenhle řádek nutný? join nepřidává oddělovač za poslední prvek seznamu. Plus ty první dva řádky bych sloučil do jednoho

`            roles = ", ".join([group.name for group in groups])`